src/ui-test/camelk_java_debug_test.ts

```diff
@@ -0,0 +1,197 @@
+			await startIntegration(INTEGRATION_LABEL);
+		})
+
+		it('Check Java Debug available', async function () {
+			this.timeout(20000);
+
+			const item = await getIntegration(INTEGRATION_LABEL);
+			const menu = await item.openContextMenu();
+			await promiseSucceed(VSBrowser.instance.driver.wait(() => menu.hasItem(START_DEBUG_LABEL), 5000));
+		});
+
+		after(async function() {
+			this.timeout(20000);
+			await removeIntegration(INTEGRATION_LABEL);
+			await prepareEmptyTestFolder(WORKSPACE_FOLDER);
+		});
+
+	});
+
+	describe('No Java Debug on Invalid Files', function() {
+
+		const INTEGRATION_LABEL = 'java-debug-test-invalid';
+		const INTEGRATION_FILE = 'JavaDebugTestInvalid';
+
+		before(async function (){
+			this.timeout(60000);
+			await moveToExplorerActivity();
+			await createIntegration(INTEGRATION_FILE);
+			await modifyCurrentFileToBeInvalid();
+			await startIntegration(INTEGRATION_LABEL);
+		});
+
+		it('Test Java Debugger Not Available On Invalid File', async function() {
+			this.timeout(20000)
+			const item = await getIntegration(INTEGRATION_LABEL);
+			const menu = await item.openContextMenu();
+			await promiseFail(VSBrowser.instance.driver.wait(() => menu.hasItem(START_DEBUG_LABEL), 5000));
+		});
+
+		after(async function() {
+			this.timeout(20000);
+			await removeIntegration(INTEGRATION_LABEL);
+			await prepareEmptyTestFolder(WORKSPACE_FOLDER);
+		});
+	})
+
+});
+
+async function prepareTempWorkspaceForTests(workspaceFolder: string) {
+	await new EditorView().closeAllEditors();
+	await prepareEmptyTestFolder(workspaceFolder);
+	await VSBrowser.instance.openResources(workspaceFolder);
+}
+
+async function camelKToolingIsEnabled() {
+	const marketplace = await Marketplace.open();
+	const item = await marketplace.findExtension(`@installed ${pjson.displayName}`);
+	return extensionIsActivated(item);
+}
+
+async function moveToExplorerActivity() {
+	const ACTIVITY_LABEL = 'Explorer';
+
+	return await moveToActivity(ACTIVITY_LABEL);
+}
+
+async function moveToActivity(viewIdentifier: string) {
+	const control = await new ActivityBar().getViewControl(viewIdentifier);
+	return VSBrowser.instance.driver.wait(async () => control?.openView());
+}
+
+async function getCamelKIntegrationSection() {
+	const section = await new SideBarView().getContent().getSection(uiTestConstants.extensionName) as CustomTreeSection;
+	await section.expand();
+	await VSBrowser.instance.driver.wait(async () => section.isExpanded());
+	return section;
+}
+
+async function getIntegration(integrationLabel: string) {
+	const section = await getCamelKIntegrationSection();
+	await VSBrowser.instance.driver.wait(async () => hasIntegration(integrationLabel));
+	const item = await section.findItem(integrationLabel) as ViewItem;
+	return item;
+}
+
+async function hasIntegration(integrationLabel: string) {
+	const section = await getCamelKIntegrationSection();
+	const isOnView = await sectionHasItem(section, integrationLabel);
+	return isOnView;
```

Comment:
can return directly isOnView
```suggestion
	return await sectionHasItem(section, integrationLabel);
```